src/write.c

```diff
@@ -2591,21 +2591,25 @@ static avifResult avifEncoderWriteMiniBox(avifEncoder * encoder, avifRWStream *
         hasExplicitCodecTypes = AVIF_FALSE;
     }
 
-    uint32_t smallDimensionsFlag = image->width <= (1 << 7) && image->height <= (1 << 7);
-    const uint32_t codecConfigSize = 4; // 'av1C' always uses 4 bytes.
+    uint32_t largeDimensionsFlag = image->width > (1 << 7) || image->height > (1 << 7); // _minus1
```

Comment:
Optional: The `// _minus1` comment may be unclear to people unfamiliar with the code. I suggest changing the comment to `// _minus1 is encoded`.

I wonder if we should rewrite the conditions to use `image->width - 1` and `image->height - 1` instead (this requires verifying that `image->width` and `image->height` are nonzero):

```
    uint32_t largeDimensionsFlag = image->width - 1 >= (1 << 7) || image->height - 1 >= (1 << 7);
```

If you make this change, make the same change to the conditions with a `// _minus1` comment below.

Reply:
> I wonder if we should rewrite the conditions to use `image->width - 1` and `image->height - 1` instead (this requires verifying that `image->width` and `image->height` are nonzero):
> 
> ```
>     uint32_t largeDimensionsFlag = image->width - 1 >= (1 << 7) || image->height - 1 >= (1 << 7);
> ```
> 
> If you make this change, make the same change to the conditions with a `// _minus1` comment below.

Done.